ibu-imager/ops/execute.go

```diff
@@ -67,7 +72,11 @@ func NewNsenterExecutor(logger *logrus.Logger, verbose bool) Execute {
 	return &nsenterExecutor{executor: executor{logger, verbose}}
 }
 
-func (e *nsenterExecutor) Execute(command string, args ...string) (string, error) {
+func (e *nsenterExecutor) ExecuteWithLiveLogger(command string, args ...string) (string, error) {
```

Comment:
Something is off here.
you have `ExecuteWithLiveLogger` twice.
Also, why wrap `Execute` with `baseExecute`?

Reply:
cause this is interface with 2 functions? and here we have 3 different options of this interface. chroot, nsenter and os.cmd